modules/basegl/src/processors/background.cpp

```diff
@@ -251,20 +262,25 @@ void Background::updateShaderInputs() {
         size_t outputLocation = 2;
 
         StrBuffer buff;
-        buff.append("\n");
-        for (size_t i = 1; i < numColorLayers; ++i) {
-            buff.append("layout(location = {}) out vec4 FragData{};\n", outputLocation++, i);
+        for (size_t i = 1; i < numColorLayers - 1; ++i) {
+            buff.append("layout(location = {}) out vec4 FragData{}; \\\n", outputLocation++, i);
         }
-        for (size_t i = 1; i < numColorLayers; ++i) {
-            buff.append("uniform sampler2D color{};\n", i);
+        for (size_t i = 1; i < numColorLayers - 1; ++i) {
```

Comment:
What about the last color layer?

Reply:
it added below... need to add the \ at the end unless on the last line...